scene/gui/color_picker.cpp

```diff
@@ -533,6 +554,7 @@ ColorPicker::ColorPicker() :
 	edit_alpha = true;
 	text_is_constructor = false;
 	raw_mode_enabled = false;
+	deferred_mode_enabled = Engine::get_singleton()->is_editor_hint();
```

Comment:
how about changing a modulate color of a node for game on editor, not for the editor theme?
I'm expecting to update it immediately while dragging.
does it work as I'm expecting?

Reply:
Ok, this settings will not apply to inspector property anymore, it will only affects editor settings.